packages/dds/tree/src/test/utils.ts

```diff
@@ -1442,3 +1452,29 @@ export function chunkToMapTreeField(chunk: TreeChunk): ExclusiveMapTree[] {
 export function nodeCursorsFromChunk(trees: TreeChunk): ITreeCursorSynchronous[] {
 	return mapCursorField(trees.cursor(), (c) => c.fork());
 }
+
+/**
+ * Construct field cursor from content that is compatible with the field defined by the provided `schema`.
+ * @param schema - The schema for what to construct. As this is an {@link ImplicitFieldSchema}, a {@link FieldSchema}, {@link TreeNodeSchema} or {@link AllowedTypes} array can be provided.
```

Comment:
Nit: I'm assuming this was copied, but the second sentence in this comment seems unnecessary (it's captured by the type-system)
```suggestion
 * @param schema - The schema for what to construct.
```